src/store/actions.ts

```diff
@@ -290,3 +290,24 @@ export const submitMarketOrder = (amount: BigNumber, side: OrderSide) => {
         }
     };
 };
+
+export const updateStore = () => {
+    return async (dispatch: any) => {
+        const web3Wrapper = await getWeb3Wrapper();
+
+        if (web3Wrapper) {
+            const [ethAccount] = await web3Wrapper.getAvailableAddressesAsync();
+            const networkId = await web3Wrapper.getNetworkIdAsync();
+
+            const knownTokens = getKnownTokens(networkId);
+
+            const tokenBalances = await Promise.all(
+                knownTokens.getTokens().map(token => tokenToTokenBalance(token, ethAccount)),
+            );
+
+            dispatch(getAllOrders());
+            dispatch(getUserOrders());
+            dispatch(setTokenBalances(tokenBalances));
```

Comment:
Can we also update the eth and weth balance?

Reply:
it's done